### PR TITLE
docs(): socket update to 0.8+ recipe

### PR DESCRIPTION
**Pull request by maciejkorsan**

### CODE REVIEW

File: docs/cookbook/web/basics/update.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+```sh
+#!/usr/bin/env sh
+
+mkdir -p .dist
+cp package.json .dist
+cd .dist
+yarn --production
+
+```
+
```
Comment (mkucharz) [inline docs/cookbook/web/basics/update.md]: too much space?

File: docs/cookbook/web/basics/update.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+At first, you have to create directory called `bin` inside of your socket. After that, place two files inside: `compile` and `compile-env` (note, that they have no extensions).
+
+`compile` file contents:
+
+```sh
+#!/usr/bin/env sh
+
+mkdir -p .dist
+babel src --out-dir .dist/src --copy-files
```
Comment (mkucharz) [inline docs/cookbook/web/basics/update.md]: here we should have `npx` I think

File: docs/cookbook/web/basics/update.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+`compile-env` file contents:
+
+```sh
+#!/usr/bin/env sh
+
+mkdir -p .dist
+cp package.json .dist
+cd .dist
+yarn --production
```
Comment (mkucharz) [inline docs/cookbook/web/basics/update.md]: maybe we should use `npm` instead yarn?

File: docs/cookbook/web/basics/update.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+```
+
+
+#### Update package.json
+
+You have to add scripts section and dependencies to your socket.
+
+```js 
+  "scripts": {
+    "build": "yarn && yarn run build:src && yarn run build:env",
```
Comment (mkucharz) [inline docs/cookbook/web/basics/update.md]: `yarn` is not dependency anymore so maybe we should use `npm`?

File: docs/cookbook/web/basics/update.md (hunk 3 above)
Comment (maciejkorsan) [reply docs/cookbook/web/basics/update.md]: it doesn't work if i change it to npm

Comment (mkucharz) [reply docs/cookbook/web/basics/update.md]: @maciejkorsan you need `npm install` (which is default in `yarn`)

**Comment by maciejkorsan** [comment]
@mkucharz changes applied

**Pull request merged**